android/history.md

```diff
@@ -1,12 +1,13 @@
 # Keyman for Android
 
-## 2019-01-11 11.0.2055 beta
-* Fix keyboard version comparison that was causing "Unable to contact Keyman server" notifications (#1520)
+## 2019-01-14 11.0.2055 beta
+* Update minimum Android SDK version to 16 (#1532)
```

Comment:
This should probably be more user-focused and say 'Keyman for Android 11 requires a minimum version of Android 4.1 (Jelly Bean)`